packages/ui/src/components/userProfile/userProfile.styled.js

```diff
@@ -0,0 +1,12 @@
+import styled from 'styled-components';
+import { Button } from 'antd';
+import { colors } from '../../../src';
+
+export const UserButton = styled(Button)`
+  display: flex;
+  justify-content: center;
+  align-items: center;
```

Comment:
Could use `mixins.flexCenter()` to achieve the same effects as the above three lines.